backend/static-db/src/users/dtos/post/create.dto.ts

```diff
@@ -0,0 +1,15 @@
+import { IsNotEmpty, IsString, IsNumber, IsDateString } from 'class-validator';
+
+export class CreatePostDTO {
+  @IsNotEmpty()
+  @IsNumber()
+  readonly moneyGoal: number;
+
+  @IsString()
+  @IsNotEmpty()
+  readonly description: string;
+
+  @IsDateString()
+  @IsNotEmpty()
+  readonly dateGoal: string;
```

Comment:
dateGoal should be nullable, since a business user may not have a particular date before the post is closed.

Reply:
Done.